index.js

```diff
@@ -162,6 +164,87 @@ const inventoryCommand = (message) => {
   });
 };
 
+const lookUpCommand = (message) => {
+  const originalMessage = message.content;
+  const fuzzySearchKey = originalMessage.slice(8);
+
+  axios
+    .get(
+      "https://db.ygoprodeck.com/api/v7/cardinfo.php?&fname=" + fuzzySearchKey
+    )
+    .then(function (cardSearchResults) {
+      console.log(
+        "https://db.ygoprodeck.com/api/v7/cardinfo.php?&fname=" + fuzzySearchKey
+      );
+
+      let searchedCardName;
+      let closestDataIndex = -2;
+      let secondClosestDataIndex;
+      cardSearchResults.data.data.forEach((element, index) => {
```

Comment:
Instead of storing indices, this might be smaller if you store the card themselves and use `.find()`.

So something like `const exactMatchCard = cardSearchResults.data.data.find(` and `const closestMatchCard = cardSearchResults.data.data.find(` instead of `closestsDataIndex` and `secondClosestDataIndex`.

This does make 1 `forEach(` become 2 `find(`s, but the 2 `find(`s are smaller and easier to understand.